src/main/NWN/API/Object/NwStationary.cs

```diff
@@ -111,5 +113,27 @@ public override float Rotation
     }
 
     public override Location Location => Location.Create(Area, Position, Rotation);
+
+    /// <summary>
+    /// Sets the saving throw value for a door or placeable object.
+    /// Must be between 0 and 250.
+    /// </summary>
+    public void SetBaseSavingThrow(SavingThrow savingThrow)
+    {
+      switch (savingThrow)
+      {
+        case SavingThrow.Fortitude:
+          NWScript.SetFortitudeSavingThrow(this, (int)savingThrow);
+          break;
+        case SavingThrow.Reflex:
+          NWScript.SetReflexSavingThrow(this, (int)savingThrow);
+          break;
+        case SavingThrow.Will:
+          NWScript.SetWillSavingThrow(this, (int)savingThrow);
+          break;
+        default:
+          throw new ArgumentOutOfRangeException(nameof(savingThrow), savingThrow, null);
+      }
```

Comment:
Need a second parameter for the actual saving throw value. This currently would not work.

Reply:
```
    public void SetBaseSavingThrow(SavingThrow savingThrow)
```
or
```
          NWScript.SetFortitudeSavingThrow(this, (int)savingThrow);
```
I'm not sure how it will not work? `NWScript.SetFortitudeSavingThrow(this, (int)savingThrow);` gets `(int)savingThrow` from `public void SetBaseSavingThrow(SavingThrow savingThrow)`?

Reply:
```cs
    /// <summary>
    /// Sets the saving throw value for a door or placeable object.
    /// (amount) must be between 0 and 250.
    /// </summary>
    public void SetBaseSavingThrow(SavingThrow savingThrow, int amount)
    {
      switch (savingThrow)
      {
        case SavingThrow.Fortitude:
          NWScript.SetFortitudeSavingThrow(this, amount);
          break;
        case SavingThrow.Reflex:
          NWScript.SetReflexSavingThrow(this, amount);
          break;
        case SavingThrow.Will:
          NWScript.SetWillSavingThrow(this, amount);
          break;
        default:
          throw new ArgumentOutOfRangeException(nameof(savingThrow), savingThrow, null);
      }
    }
```